site/docs/advanced/transformers.md

```diff
@@ -51,7 +51,7 @@ Calls via context objects of concurrently running middleware are left unaffected
 As soon as the respective middleware completes, the transformer function is discarded.
 
 ```ts
-bot.on("message", (ctx) => {
+bot.on("message", async (ctx) => {
```

Comment:
We should call next here

Reply:
In this pull request, or should we do this in another?

Reply:
In a subsequent PQ. It's kinda unrelated.